universions/version.py

```diff
@@ -15,6 +15,71 @@ class Version(NamedTuple):
     prerelease: Optional[str] = None
     build: Optional[str] = None
 
+    def __lt__(self, other: Any) -> bool:
+        if not isinstance(other, tuple):
+            return NotImplemented
```

Comment:
Isn't it a problem that the operation is not symmetrical?
I am quite a novice in Python so I don't know the philosophy, but it does not seem very mathematical/logical

Reply:
Why isn't it symmetrical ?